### DISCUSSION TITLE
Error while running `dotnet publish <..>`

### DISCUSSION THREAD

**Discussion by verma-kartik** [ForNeVeR/Cesium#554] [General]
I'm running `dotnet publish Cesium.Compiler/Cesium.Compiler.csproj -r osx-arm64 --self-contained` and it is giving me `error : PublishAot and PublishSingleFile cannot be specified at the same time.` error.

Also, the [doc ](https://github.com/ForNeVeR/Cesium/blob/main/CONTRIBUTING.md)says to install .NET SDK 8.0 or later, however when the project is initialised in VSCode, it automatically installs .NET Runtime 7.0.16. 

Is the above error due to version mismatch? or something else? 

PS: For other OS, the `-r` flag in `dotnet publish` needs to updated in the docs.

**Comment by BadRyuner** [comment]
Different parts of the project are targeting Net6 or Net7 or Net8. So all three SDKs are recommended (I think). Also the PublishSingleFile flag is not needed, because it is already defined in the cesium.compiler.csproj file.

**Reply by verma-kartik** [reply]
It worked by commenting out PublishSingleFile flag in cesium.compiler.csproj.

**Comment by ForNeVeR** [comment]
I believe this is a bug in our documentation, and perhaps in the project files as well. Opened an issue #555 on this. Thanks for reporting!
